node-js-web-server/server050.js

```diff
@@ -0,0 +1,13 @@
+const http = require('http');
+
+http.createServer(function (request, response) {
+   
+}).listen(8080);
+
+// now you can run your server by typing `node server` in your terminal
```

Comment:
We won't actually be able to run a server here, so we'll have to modify this text a bit:

While there's no way to run your server from this page, you could run it locally by saving it to a file called `server.js` and running `node server` from the same directory in your terminal.
However, if you did, there would be no way of knowing that your server is actually running.
Use `console.log()` to write your own notification that the server is running at port 8080.